src/platform/mac/macplatformwindow.mm

```diff
@@ -129,17 +129,19 @@ void sendShortcut(int modifier, int key, pid_t pid = -1) {
         CGEventSetFlags(VUp,CGEventFlags(kCGEventFlagMaskCommand|0x000008));
 
         if (pid != -1) {
-            CGEventPostToPid(pid, commandDown);
-            CGEventPostToPid(pid, VDown);
-            CGEventPostToPid(pid, VUp);
-            CGEventPostToPid(pid, commandUp);
-        } else {
-            CGEventPost(kCGHIDEventTap, commandDown);
-            CGEventPost(kCGHIDEventTap, VDown);
-            CGEventPost(kCGHIDEventTap, VUp);
-            CGEventPost(kCGHIDEventTap, commandUp);
+	    COPYQ_LOG( QStringLiteral("******* Not using CGEventPostToPid : %1").arg(pid));
+            /* CGEventPostToPid(pid, commandDown); */
+            /* CGEventPostToPid(pid, VDown); */
+            /* CGEventPostToPid(pid, VUp); */
+            /* CGEventPostToPid(pid, commandUp); */
         }
 
+        COPYQ_LOG( QStringLiteral("******* Using CGEventPost"));
```

Comment:
It's probably better to remove this completely (even with the `pid` argument), the logs nor the commented-out code look particularly useful.

Reply:
Done